### PR TITLE
feat: add programming languages

### PR DESCRIPTION
**Pull request by Technologeek**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngneat/falso/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #7 

## What is the new behavior?
Added programming languages falso.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by Technologeek** [comment]
@NetanelBasal Do we need tests for this?

File: packages/falso/src/lib/programming-languages.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { rand } from './core';
+
+export function programmingLanguages() {
```
Comment (iamandrewluca) [inline packages/falso/src/lib/programming-languages.ts:3]: I would suggest to rename this to singular
```suggestion
export function programmingLanguage() {
```

**Review by iamandrewluca — CHANGES_REQUESTED** [review]

**Comment by iamandrewluca** [comment]
@Technologeek I think there are no need for tests for functions that just return random data from an array

**Comment by Technologeek** [comment]
@iamandrewluca Thank you! I've addressed your comments in the new commit.

File: packages/falso/src/lib/programming-language.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { rand } from './core';
+
+export function programmingLanguage() {
+  return rand([
```
Comment (NetanelBasal) [inline packages/falso/src/lib/programming-language.ts:4]: Can you please migrate to the new way using a JSON file?

Comment (Technologeek) [reply packages/falso/src/lib/programming-language.ts]: @NetanelBasal PR updated thanks!

**Review by iamandrewluca — APPROVED** [review]

**Pull request merged**